cli/diffParser.js

```diff
@@ -84,16 +82,22 @@ function convertToDiffObject(granule) {
         //to both the old and new versions
         var oldID = null;
         var newID = null;
-        granule.versions.old.attributes.forEach(function(attribute){
-           if(attribute.name == 'por-id') {
-               oldID = attribute.value;
-           }
-        });
-        granule.versions.new.attributes.forEach(function(attribute){
-            if(attribute.name == 'por-id') {
-                newID = attribute.value;
-            }
-        });
+
+        if('attributes' in granule.versions.old) {
+            granule.versions.old.attributes.forEach(function (attribute) {
+                if (attribute.name == 'por-id') {
+                    oldID = attribute.value;
```

Comment:
Should be able to `return` to break out of the `forEach` early, now that you have what you're looking for.